packages/app/basicStyles.css

```diff
@@ -0,0 +1,19 @@
+html {
```

Comment:
You could put that into the old: `packages/app/src/index.css`

Reply:
yep. Too many css files for the app, indeed!

Reply:
fixed in c9562eec904cabada1f650c950ed9ae88aacd317